### DISCUSSION TITLE
Depremzede Çoçuklar için Body Süreci

### DISCUSSION THREAD

**Discussion by hasaneks** [acikkaynak/deprem-yardim-projesi#27] [Ideas]
Merhaba, 

Biraz önce Cüneyt Özdemir'in yayınında bir fikir ortaya atıldı. Bu fikir bence ilerleyen süreç için çok fazla katkı sağlayabilir.

- Depremzede çocukların eğitim masraflarını üstlenebileceğiniz ya da deprem zede bir çoçuk ile eşleşebileceğiniz bir uygulama.

Sadece maddiyat değil aynı zamanda psikolojik olarakta çok faydalı olabileceğini düşünüyorum. 

Bu tarz bir proje için süreç nasıl yapılabilir, ya da böyle bir proje şuan için planlanmakta mı ?

Not : React & React Native konusunda yardımcı olabilirim.

**Comment by kivilcimcagatay** [comment]
Aynı şeyi gruba yazmıştım ben de. Kesinlikle çok önemli konulardan bir tanesi. UI UX Research anlamında destek olmaya hazırım

**Reply by hasaneks** [reply]
Özür dilerim ben görmemiştim. Cüneyt Özdemir'in yayınında görünce mantıklı gelince paylaşmak istedim.

**Reply by kivilcimcagatay** [reply]
yok ne özürü ben de yazıldı mı diye bakıyordum

**Reply by alihansencan** [reply]
harika bir fikir.

**Comment by ErhanArda** [comment]
kesinlikle cok iyi fikir ama cocuklarin isimleri ya da bilgileri gorunur bir sekilde olmamasi gerekir diye dusunuyorum
once bu konuda destek oiabilecek insanlarin bir bilgilerini biryerde toplanmasi gerekir, kimlerin hangi konularda destek olabilecekleri bilgisi tutulmali buan gore cocuklarin ihtiyac dagilimi olmali diye dusunuyorum

**Reply by berkAktug** [reply]
Parasal yardım dışında hangi kategoriler olması gerekir dersin? benim aklıma gelmedi

**Reply by ErhanArda** [reply]
- egitim 
- kiyafet
- saglik
suan aklima gelenler ama cesitlendirilebilir

**Reply by kivilcimcagatay** [reply]
psikolojik destek. şu an yaşanılanlar çok büyük travma bunun için destek lazım.

**Reply by ErhanArda** [reply]
evimdekipsikolog gibi online sitelerden yardim istenebilir
her cocuga yuz yuze destek olmak suan zor ama online olarak surecte destek olunabilir diye dusunuyorum

**Reply by kivilcimcagatay** [reply]
pedagog olması önemli söylediğiniz yerde var mı

**Reply by ErhanArda** [reply]
cocuk ve erken psikiyatrisi var ki ekstra durumlar icin onlarin yonlendirmesi ile hareket edilebilir
buralar ile de iletisime gecilebilir
http://pedagogonline.com/
https://kidolog.com/

**Reply by ErhanArda** [reply]
evimdekipsikolog depremzedeler icin ucretsiz destek saglamis

**Comment by timurturbil** [comment]
Eser ve berfu yenenler hali hazırda bazı çocuklara sponsor olmuşlar, proje geliştirmeye başlanırsa onlarla da iletişime geçilirse proje daha çok destek alabilir.

**Reply by hasaneks** [reply]
Aynen abi, bende oradan gördüm zaten. Bu tarz projeleri zaten hem STK'lar ile hemde dediğin gibi bazı gönüllü kişiler ile beraber yürütmek çok daha sağlıklı olacaktır.

**Comment by berkAktug** [comment]
Backend konusunda gerek Spring-Spring Boot gerek C# .NET ile yardım etmeye hazırım. 
Burda açıkcası çocuklardan bağımsız şekilde Mentor kişilerin (Masrafları üstlenecek kişi) profillemesine gidilebilir diye düşünüyorum
Discord:Berk Aktuğ#9039

**Comment by sercangoger** [comment]
en başta bende mobil olarak düşünüyorum bu tarz işleri aslında ama buna gerek olacağını düşünmüyorum. daha hızlı bir şekilde şuan deprem.io içerisinde bir sayfa yaparak oluşturulabilir.

**Reply by berkAktug** [reply]
Aslında deprem.io'ya entegre etmek bunları daha mantıklı olabilir ama siteyi çok yüklemiş olur muyuz? Tek çatıda toplamak daha mantıklı da olabilir.

**Reply by sercangoger** [reply]
deprem.io içerisinde olması çok büyük birşey olmayacağına inanıyorum. amaç aslında deprem.io platformunun büyültülüp gerçek ihtiyacların tek bir yerden olması daha akılcı ve doğru olacaktır.

**Comment by furkan-cat** [comment]
Sosyal hizmetçi birinden bu konuda bilgi alınması gerek diye düşünüyorum. Bir çocuğa kontak kurmadan maddi bir destek sunmak okey ama kontak kurup sahip çıkıp korumak kollamak durumu beraberinde bir çok parametrenin de süzgeçten geçmesi demek. Sahip çıkmak isteyen kişinin psikolojik durumu, maddi durumu, hırlı mı hırsız mı kontrol edilmesi gereken bir sürü parametre olsa gerek ki sosyal hizmet uzmanları hali hazırda bu süreçleri aylarca araştırarak yerine getiriyor. Bunun gerçekleştirebilirliğini ilk etapta sosyal hizmetçi birilerinden öğrenilmesi gerektiğini düşünüyorum. Olayın bir de kanunsal boyutu da var. O da ayrı bir konu.

**Comment by alihansencan** [comment]
Arkadaslar cocuklara gönüllü olarak egitim verebilmemiz acisindan da cok iyi bir proje olacagini düsünüyorum. derslerine yardimci olmak ve yetkin oldugumuz konularda da gencleri ve cocuklari egitip destek olunabilir. ne kadar bir öncelik siralamasi olur dedigimin bilmiyorum ama.

**Comment by alihansencan** [comment]
Backend JavaScript , TypeScript ve Frontend 'de Vue.JS tarafinda yardimci olabilirim.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Legal ekibi projeyi değerlendi. Ve şu notları iletti.

"Bu proje ancak bakanlık ile entegre ilerletilirse mümkün aksi durumda legale takılır"

**Comment by TolgaTaner** [comment]
Mobile tarafta native kararı verilirse, iOS kısmı için destek olurum

**Comment by DID0101** [comment]
merhabala iki taraftada yardimci labilirim MERN stack da daha iyiim yardimci olmak istiyorum

**Comment by dorukane2** [comment]
Arkadaşlar selamlar. Dünden beri bağlantı problemleri yaşadığım için bu konuyu çok ilerletemedim ama Discord'da da bunu kısaca konuşmaya başlamıştık.

Bu projeyi legale takılmayacak bir hale getirebiliriz belki. Bir çocuğu yetkinliği doğrulanmamış herhangi biriyle doğrudan eşleştirmek hatta iletişim kurmalarını sağlamak baya sakıncalı durumlara yol açabilir.

Ancak, anonim bir eşleştirme o kadar sıkıntılı olmayabilir. Kim kim bilmeden, bir kişi bir çocuğun eğitim masraflarını üstlenebilir diye düşünüyorum. Aylık Spotify aboneliği satın alır gibi bir model. Bu tarafta platform sadece eğitim masrafları için bağış toplamış oluyor. Bunu bu şekilde yine legalden geçirip geçiremeyeceğimizi bi öğrenmek isterim :) @ramazansancar 

Asıl iş diğer tarafta tabi yapılabilecek şey çok. Çağdaş Eğitim Vakfı, Darüşşafaka gibi eğitim kurumlarıyla ihtiyaç sahibi çocukların eğitimleri yönetilebilir. Çocuklara özel destek hizmeti sağlayacak profesyonel sosyal hizmetçilerden oluşan bir ekip kurularak çocuklara düzenli destek sağlanılabilir. Çocukların birbirleriyle iletişime geçebileceği bir alan olabilir vs.
